girder/api/v1/group.py

```diff
@@ -251,7 +263,25 @@ def inviteToGroup(self, group, params):
             id=params['userId'], user=user, level=AccessType.READ, exc=True)
 
         if force:
-            self.requireAdmin(user)
+            if not user.get('admin', False):
+                mustBeAdmin = True
+                addPolicy = self.model('setting').get(
+                    SettingKey.ADD_TO_GROUP_POLICY)
+                addGroup = group.get('addAllowed', 'default')
+                if addGroup not in ['no', 'yesadmin', 'yesmod']:
```

Comment:
Should we turn these strings into constants? I don't have an opinion one way or the other, just bringing up the dogmatic position.